### PR TITLE
Fix reference dir

### LINKED ISSUE #166: Could not set unknown property 'referenceDir'
cannot set custom directory for screenshots, i get the following error,
"Could not set unknown property 'referenceDir' for object of type com.facebook.testing.screenshot.build.ScreenshotsPluginExtension."

`screenshots {
    referenceDir = "$projectDir/build/reports/screenshots"
    targetPackage = "com.x.x"
}`

### PR DESCRIPTION
**Pull request by tiagoengel**
Add back the logic to only verify screenshots and not record them when referenceDir is present. 

**NOTE**: I can't make the current project I'm working on work on version 9 (or current master) because of some internal error finding a resource. Would you be open to release a new version for 8 with this fix too? I can send a PR for that branch too, once this is merged.

fixes #166

### CODE REVIEW

File: .idea/codeStyleSettings.xml [hunk 1]
```diff
@@ -1,251 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```
Comment (xiphirx) [inline .idea/codeStyleSettings.xml]: Dont delete this please.

File: plugin/src/main/kotlin/com/facebook/testing/screenshot/build/ScreenshotsPlugin.kt [hunk 2]
```diff
@@ -32,6 +32,8 @@ open class ScreenshotsPluginExtension {
   var multipleDevices = false
   /** The python executable to use */
   var pythonExecutable = "python"
+  /** The directory to pull screenshots from in verify only mode */
```
Comment (xiphirx) [inline plugin/src/main/kotlin/com/facebook/testing/screenshot/build/ScreenshotsPlugin.kt]: Lets change this to
```suggestion
  /** The directory to compare screenshots from in verify only mode */
```

File: .idea/codeStyleSettings.xml (hunk 1 above)
Comment (tiagoengel) [reply .idea/codeStyleSettings.xml]: Sorry, didn't realise this. Done

File: plugin/src/main/kotlin/com/facebook/testing/screenshot/build/ScreenshotsPlugin.kt (hunk 2 above)
Comment (tiagoengel) [reply plugin/src/main/kotlin/com/facebook/testing/screenshot/build/ScreenshotsPlugin.kt]: Done

**Comment by tiagoengel** [comment]
Sure, no big deal about the current master. I'll keep using my fork in the meantime and try to help when I have some free time. 

I've addressed all the comments 🙂

**Pull request closed without merging**